lib/Parse/ParseExpr.cpp

```diff
@@ -2621,8 +2622,11 @@ ParserStatus Parser::parseClosureSignatureIfPresent(
         // It is a common error to try to capture a nested field instead of just
         // a local name, reject it with a specific error message.
         if (Tok.isAny(tok::period, tok::exclaim_postfix,tok::question_postfix)){
-          diagnose(Tok, diag::cannot_capture_fields);
-          skipUntil(tok::comma, tok::r_square);
+          auto diag = diagnose(Tok, diag::cannot_capture_fields);
+          skipTo(tok::comma, tok::r_square);
```

Comment:
This naming confusable with `skipUntil`. Since this is the only usage at this point, could you just inline this?
```
while (peekToken().isNot(tok::comma, tok::r_square, tok::eof, tok::pound_endif, 
                         tok::pound_else, tok::pound_elseif))
  skipSingle()
````
This doesn't look too much noise.

Reply:
done